agent/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       - citamon-agent-net
 
   citamon_agent_process_exporter:
-    image: ncabatoff/process-exporter:0.4.0
+    image: citamon/agent-process-exporter:0.4.0
```

Comment:
we need to get repo "citamon/agent-process-exporter" source opened, put it into a sub dir